contrib/win32/win32compat/wmain_sshd.c

```diff
@@ -129,7 +129,7 @@ generate_host_keys()
 		ui.usri1_priv = USER_PRIV_USER;
 		ui.usri1_home_dir = NULL;
 		ui.usri1_comment = NULL;
-		ui.usri1_flags = UF_SCRIPT;
+		ui.usri1_flags = UF_SCRIPT | UF_DONT_EXPIRE_PASSWD;
```

Comment:
I was unable to find a way to similar an account with an expired password. As per msdn, there is no way to force a password expiry on an existing account. 

Otherwise, password expiry shouldn't impact us, since we are using s4u to generate tokens. I added this just so we are doing the right thing. Also, with the new virtual account token changes in, we should never generate sshd account tokens (we still need sshd account to comply with Unix code and have a fallback mechanism in virtual account generation fails).